internal/mode/static/nginx/config/stream_servers.go

```diff
@@ -60,13 +66,28 @@ func createStreamServers(conf dataplane.Configuration) []stream.Server {
 		}
 
 		portSet[server.Port] = struct{}{}
-		streamServers = append(streamServers, stream.Server{
+
+		streamServer := stream.Server{
```

Comment:
Let's add a comment here explaining why we don't add the rewrite client ip settings in this case.